demo/requirements.txt

```diff
@@ -1,2 +1,4 @@
-doctr
+-e git+https://github.com/mindee/doctr.git#egg=doctr
+tensorflow
+tensorflow-addons
```

Comment:
Thanks!
In the same spirit, I would suggest so that it stays robust to future evolutions of the dependencies:
```
-e git+https://github.com/mindee/doctr.git#egg=doctr[tf]
```

Reply:
Alright, good idea! Done!